app/Http/Controllers/PagesController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Blog;
+use App\Blogcomment;
+class PagesController extends Controller
+{
+    public function home()
+     {
+         $show = Blog::all();
+         $show = Blog::paginate(4);
+        return view('welcome',compact('show'));
+     }
+
+     
+     public function addblog()
+     {
+        return view('addblog');
+     }
+
+     public function store()
+     {
+        $project = new Blog();
+        $project->title = request('title');
+        $project->blog = request('blog');
+
+        $project->save();
```

Comment:
line track should come after save (different code blocks

- Save project (Why $project, why not $page.)
- redirect